### PR TITLE
Translate ServiceList to yaml

### PR DESCRIPTION
**Pull request by openBackhaul**

### CODE REVIEW

File: AdministratorAdministration+forwardings.yaml [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+        - server-name: /v1/disregard-application
+          uuid: aa-1-0-0-op-s-is-002
+      operation-client-deletion:
+        - server-name: /v1/disregard-application
+          uuid: aa-1-0-0-op-s-is-002
+    initiating-requests:
+        - server-name: /v1/regard-application
+          uuid: aa-1-0-0-op-s-is-001
+    consequent-requests:
+      - client-name: RegistryOffice:://v1/inquire-oam-request-approvals
```
Comment (IswaryaaS) [inline AdministratorAdministration+forwardings.yaml:484]: As per my understanding, /v1/inquire-oam-request-approvals (operation-client under each application) is applicable only for AA application, changing the api-segment of all occurences of /v1/inquire-oam-request-approvals to "individual management (im)" would be more precise.

Kindly let me know your views

File: AdministratorAdministration+services.yaml [hunk 2]
```diff
@@ -30,6 +30,8 @@ servers:
           uuid: aa-1-0-0-op-s-1000
     service:
       basic:
+        - service-name: /v1/start-application-in-generic-representation
+          uuid: aa-1-0-0-op-s-2000
         - service-name: /v1/inform-about-application
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply AdministratorAdministration+services.yaml:35]: Changed as proposed.

File: AdministratorAdministration+forwardings.yaml (hunk 1 above)
Comment (openBackhaul) [reply AdministratorAdministration+forwardings.yaml:484]: Changed as proposed.

File: AdministratorAdministration+services.yaml [hunk 3]
```diff
@@ -30,6 +30,8 @@ servers:
           uuid: aa-1-0-0-op-s-1000
     service:
       basic:
+        - service-name: /v1/start-application-in-generic-representation
+          uuid: aa-1-0-0-op-s-2000
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply AdministratorAdministration+services.yaml:34]: Changed as proposed.

File: AdministratorAdministration+services.yaml [hunk 4]
```diff
@@ -30,6 +30,8 @@ servers:
           uuid: aa-1-0-0-op-s-1000
     service:
       basic:
+        - service-name: /v1/start-application-in-generic-representation
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply AdministratorAdministration+services.yaml:33]: Changed as proposed.

File: AdministratorAdministration+forwardings.yaml [hunk 5]
```diff
@@ -0,0 +1,499 @@
...
+    management-requests:
+      operation-client-update:
+      fc-port-update:
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+      - server-name: /v1/bequeath-your-data-and-die
+        uuid: aa-1-0-0-op-s-im-000
+    consequent-requests:
+      - client-name: RegistryOffice:://v1/end-subscription
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply AdministratorAdministration+forwardings.yaml:187]: Changed as proposed.

File: AdministratorAdministration+forwardings.yaml [hunk 6]
```diff
@@ -0,0 +1,499 @@
+forwardings:
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply AdministratorAdministration+forwardings.yaml:1]: Changed as proposed.

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: AdministratorAdministration+data.json [hunk 7]
```diff
@@ -579,7 +579,7 @@
               "aa-1-0-0-op-s-2002",
               "aa-1-0-0-op-s-2003",
               "aa-1-0-0-op-s-2004",
-              "aa-1-0-0-op-s-3000",
+              "aa-1-0-0-op-s-2000",
               "aa-1-0-0-op-s-3001",
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply AdministratorAdministration+data.json:583]: Changed as proposed. Many thanks for indicating.

File: AdministratorAdministration+forwardings.yaml [hunk 8]
```diff
@@ -0,0 +1,501 @@
...
+          uuid: aa-1-0-0-op-s-is-002
+      operation-client-deletion:
+        - server-name: /v1/disregard-application
+          uuid: aa-1-0-0-op-s-is-002
+    initiating-requests:
+        - server-name: /v1/regard-application
+          uuid: aa-1-0-0-op-s-is-001
+    consequent-requests:
+      - client-name: RegistryOffice:://v1/inquire-oam-request-approvals
+        uuid: aa-1-0-0-op-c-im-ro-1-0-0-000
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply AdministratorAdministration+forwardings.yaml:487]: Changed as proposed. Many thanks for indicating.

File: AdministratorAdministration+data.json [hunk 9]
```diff
@@ -1014,7 +1014,7 @@
             ]
           },
           {
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply AdministratorAdministration+data.json:1016]: Changed as proposed. Many thanks for indicating the mistake.

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**